### DISCUSSION TITLE
Switching to a pinned tab

### DISCUSSION THREAD

**Discussion by otonoton** [mbnuqw/sidebery#1793] [Q&A]
Is there a way to quickly switch to a pinned tab with the keyboard? I only have one pinned but I want to be able to easily switch to it without using the mouse.

**Comment by JohnVKRDam** [comment]
Hi,

You can use the Firefox default shortcuts for that:

**Ctrl + 1** = 1st (Pinned) Tab
**Ctrl + 2** = 2nd (Pinned) Tab
**Ctrl + 3** = 3rd (Pinned) Tab
...
**Ctrl + 8** = 8th (Pinned) Tab

Note that:
**Ctrl + 0** is reserved for "Page Zoom Reset"
**Ctrl + 9** is reserved for "Go To Last Tab" (which means it will switch to the VERY LAST TAB in the VERY LAST PANEL when using Sidebery)

I have included a screenshot with these notes, for easy archiving  ;)

You can find an overview of all common Firefox shortcuts here: https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly

.
.
![Firefox - Switching to Pinned Tabs using a Keyboard Shortcut](https://github.com/user-attachments/assets/bcb1fa9d-3594-4b60-a795-117e012cc181)
